lib/gitbot_backend.rb

```diff
@@ -35,17 +35,18 @@ def run_script
     puts out
   end # main function for doing the test
 
-  def pr_test(upstream, pr_sha_com, repo, pr_branch, pr)
+  def pr_test(repo, pr, https)
```

Comment:
Refactored to avoid problems at rubocop: https://travis-ci.org/openSUSE/gitbot/builds/278835768?utm_source=github_status&utm_medium=notification

Reply:
+1